content/en/blog/_posts/2022-07-27-sig-apps-GA-1.25/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: "sig-apps features graduating to stable in 1.25"
+date: 2022-07-27
+slug: "sig-apps features graduating to stable in 1.25"
+---
+
+**Authors:** Ravi Gudimetla (Apple), Filip Křepinský (Red Hat), Maciej Szulik (Red Hat)
+
+This blog describes the two features namely `minReadySeconds` for StatefulSets and `maxSurge` for DaemonSets that sig-apps is happy to graduate to stable in 1.25.
+
+Specifying `minReadySeconds` slows down a rollout of a StatefulSet when using a `RollingUpdate` `updateStrategy` by waiting for each pod for a desired time. 
+This time can be used for initializing the pod (e.g. warming up the cache) or as a delay before acknowledging the pod.
+
+`maxSurge` allows a DaemonSet workload to run multiple instances of the same pod on a node during a rollout when using a `RollingUpdate` `updateStrategy`.
+This helps to minimize the downtime of the DaemonSet to other consumers.
+
+These features are already available in a Deployment and other workloads. This graduation helps to align this functionality across the workloads.
+
+## What problems do these features solve?
+
+### MinReadySeconds for StatefulSets
+`minReadySeconds` ensures that the StatefulSet workload is `Ready` for the given number of seconds before calling the
+pod `Available`. The notion of being `Ready` and `Available` is quite important for workloads. For example, some workloads like Prometheus with multiple instances of Alertmanager should be considered `Available` only when the Alertmanager's state transfer is complete. `minReadySeconds` also helps when using loadbalancers with cloud providers. Since the pod should be `Ready` for the given number of seconds, it provides buffer time to prevent killing pods in rotation before new pods show up.
```

Comment:
```suggestion
pod `Available`. The notion of being `Ready` and `Available` is quite important for workloads. For example, some workloads, like Prometheus with multiple instances of Alertmanager, should be considered `Available` only when the Alertmanager's state transfer is complete. `minReadySeconds` also helps when using loadbalancers with cloud providers. Since the pod should be `Ready` for the given number of seconds, it provides buffer time to prevent killing pods in rotation before new pods show up.
```